gridsome.config.js

```diff
@@ -36,5 +35,13 @@ module.exports = {
         publicPath: `/admin`
       }
     }
-  ]
+  ],
+  templates: {
+    Episode: [
+      {
+        path: '/episodes/1',
```

Comment:
uh, yeah... we are going to have to fix this 😆I could not figure out how to get the episode number to increment.  so i decided to make a separate issue for it instead of holding up this branch. https://github.com/enjoythevue/website/issues/33

Reply:
Sounds great. Thank you!